Dockerfile

```diff
@@ -0,0 +1,52 @@
+# BUILD:
+# docker build --force-rm=true -t openvpn_exporter .
+
+# RUN:
+# docker run -it -p 9176:9176 -v /path/to/openvpn_server.status:/etc/openvpn_exporter/server.status openvpn_exporter
+
+FROM golang as builder
+
+# Grenoble testing env
+ENV http_proxy="http://web-proxy.gre.hpecorp.net:8080" \
+    https_proxy="http://web-proxy.gre.hpecorp.net:8080"
```

Comment:
This seems like some left-over from development, right?

Reply:
Yes I removed it already.